### DISCUSSION TITLE
Welcome to superset-ui Discussions!

### DISCUSSION THREAD

**Discussion by ktmud** [apache-superset/superset-ui#854] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by junlincc** [comment]
Is there a reason we can't have the discussion in the main repo? hm there's no Discussion tab there..

**Reply by ktmud** [reply]
You need admin access to enable this feature. We'd have to go through Apache.
